teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < person
+  attr_accessor :specialization
+
+  def initialize(specialization)
```

Comment:
- [ ] Please pass the `age`, `name = "Unknown"`, `parent_permission = true` to the initialize method so you can pass it to the superclass. :smile: